src/terra-dev-site/guide/Component Guides/RecreatingItemCollection.guide.mdx

```diff
@@ -0,0 +1,201 @@
+import ResponsiveListandTableExample from '../../example/ResponsiveListAndTableExample?dev-site-example';
+import ResponsiveListandCellGridExample from '../../example/ResponsiveListAndCellGridExample?dev-site-example';
+import ResponsiveListandHtmlTableExample from '../../example/ResponsiveListAndHtmlTableExample?dev-site-example';
+
+# Re-creating Item Collection
+
+An Item Collection is a wrapper component designed to display data as either a table or list of items dependent on the relative container size.
+
+The `terra-item-collection` used [terra-responsive-element](/components/terra-responsive-element/responsive-element/responsive-element) to swap between it's [terra-table](/components/terra-table/table/table) and [terra-list](/components/terra-list/list/list) displays. The examples in this guide use `ActiveBreakpointContext` from [terra-application](/application/terra-application/application) to switch what is displayed based on current browser size.
+
+```javascript
+import { ActiveBreakpointContext } from 'terra-application/lib/breakpoints';
+...
+const CustomItemCollection = () => {
+  const activeBreakpoint = React.useContext(ActiveBreakpointContext);
+  ...
+  switch (activeBreakpoint) {
+    case 'tiny':
+    case 'small':
+    case 'medium':
+      return listDisplay;
+    case 'large':
+    case 'huge':
+    case 'enormous':
+    default:
+      return tableDisplay;
+  }
+};
+```
+
+This will display our list display at browser size `medium` and below and our table display on larger sizes.
+
+The examples also use [terra-clinical-item-view](/components/terra-clinical-item-view/clinical-item-view/clinical-item-view) for each cell of data which is not necessary and is used to to recreate the deprecated `terra-item-collection`.
+
+## Replicating the Deprecated Terra Clinical Item Collection displays
+
```

Comment:
Add an introduction here talking about the three different methods of re-creating the item collection and why you'd choose one over the other.

Reply:
Mention that the same list is used for each of the three different methods

Reply:
Addressed here https://github.com/cerner/terra-ui/pull/310/commits/e9da608d29aaca7a6a54bf05797065b88e72b842#diff-f306b40c35b7837e14535b95656559fca5a16191b1c2d2c2d535b3e07e66705aR35